--- ../src-base/minecraft/net/minecraft/network/datasync/DataSerializers.java
+++ ../src-work/minecraft/net/minecraft/network/datasync/DataSerializers.java
@@ -32,13 +32,13 @@
     };
     public static final DataSerializer<Integer> field_187192_b = new DataSerializer<Integer>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Integer p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Integer value)
         {
-            p_187160_1_.func_150787_b(p_187160_2_.intValue());
+            buf.func_150787_b(value.intValue());
         }
-        public Integer func_187159_a(PacketBuffer p_187159_1_)
+        public Integer func_187159_a(PacketBuffer buf)
         {
-            return Integer.valueOf(p_187159_1_.func_150792_a());
+            return Integer.valueOf(buf.func_150792_a());
         }
         public DataParameter<Integer> func_187161_a(int p_187161_1_)
         {
@@ -47,13 +47,13 @@
     };
     public static final DataSerializer<Float> field_187193_c = new DataSerializer<Float>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Float p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Float value)
         {
-            p_187160_1_.writeFloat(p_187160_2_.floatValue());
+            buf.writeFloat(value.floatValue());
         }
-        public Float func_187159_a(PacketBuffer p_187159_1_)
+        public Float func_187159_a(PacketBuffer buf)
         {
-            return Float.valueOf(p_187159_1_.readFloat());
+            return Float.valueOf(buf.readFloat());
         }
         public DataParameter<Float> func_187161_a(int p_187161_1_)
         {
@@ -62,13 +62,13 @@
     };
     public static final DataSerializer<String> field_187194_d = new DataSerializer<String>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, String p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, String value)
         {
-            p_187160_1_.func_180714_a(p_187160_2_);
+            buf.func_180714_a(value);
         }
-        public String func_187159_a(PacketBuffer p_187159_1_)
+        public String func_187159_a(PacketBuffer buf)
         {
-            return p_187159_1_.func_150789_c(32767);
+            return buf.func_150789_c(32767);
         }
         public DataParameter<String> func_187161_a(int p_187161_1_)
         {
@@ -77,13 +77,13 @@
     };
     public static final DataSerializer<ITextComponent> field_187195_e = new DataSerializer<ITextComponent>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, ITextComponent p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, ITextComponent value)
         {
-            p_187160_1_.func_179256_a(p_187160_2_);
+            buf.func_179256_a(value);
         }
-        public ITextComponent func_187159_a(PacketBuffer p_187159_1_) throws java.io.IOException
+        public ITextComponent func_187159_a(PacketBuffer buf) throws java.io.IOException
         {
-            return p_187159_1_.func_179258_d();
+            return buf.func_179258_d();
         }
         public DataParameter<ITextComponent> func_187161_a(int p_187161_1_)
         {
@@ -92,13 +92,13 @@
     };
     public static final DataSerializer<Optional<ItemStack>> field_187196_f = new DataSerializer<Optional<ItemStack>>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Optional<ItemStack> p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Optional<ItemStack> value)
         {
-            p_187160_1_.func_150788_a((ItemStack)p_187160_2_.orNull());
+            buf.func_150788_a((ItemStack)value.orNull());
         }
-        public Optional<ItemStack> func_187159_a(PacketBuffer p_187159_1_) throws java.io.IOException
+        public Optional<ItemStack> func_187159_a(PacketBuffer buf) throws java.io.IOException
         {
-            return Optional.<ItemStack>fromNullable(p_187159_1_.func_150791_c());
+            return Optional.<ItemStack>fromNullable(buf.func_150791_c());
         }
         public DataParameter<Optional<ItemStack>> func_187161_a(int p_187161_1_)
         {
@@ -107,20 +107,20 @@
     };
     public static final DataSerializer<Optional<IBlockState>> field_187197_g = new DataSerializer<Optional<IBlockState>>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Optional<IBlockState> p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Optional<IBlockState> value)
         {
-            if (p_187160_2_.isPresent())
+            if (value.isPresent())
             {
-                p_187160_1_.func_150787_b(Block.func_176210_f((IBlockState)p_187160_2_.get()));
+                buf.func_150787_b(Block.func_176210_f((IBlockState)value.get()));
             }
             else
             {
-                p_187160_1_.func_150787_b(0);
+                buf.func_150787_b(0);
             }
         }
-        public Optional<IBlockState> func_187159_a(PacketBuffer p_187159_1_)
+        public Optional<IBlockState> func_187159_a(PacketBuffer buf)
         {
-            int i = p_187159_1_.func_150792_a();
+            int i = buf.func_150792_a();
             return i == 0 ? Optional.<IBlockState>absent() : Optional.of(Block.func_176220_d(i));
         }
         public DataParameter<Optional<IBlockState>> func_187161_a(int p_187161_1_)
@@ -130,13 +130,13 @@
     };
     public static final DataSerializer<Boolean> field_187198_h = new DataSerializer<Boolean>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Boolean p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Boolean value)
         {
-            p_187160_1_.writeBoolean(p_187160_2_.booleanValue());
+            buf.writeBoolean(value.booleanValue());
         }
-        public Boolean func_187159_a(PacketBuffer p_187159_1_)
+        public Boolean func_187159_a(PacketBuffer buf)
         {
-            return Boolean.valueOf(p_187159_1_.readBoolean());
+            return Boolean.valueOf(buf.readBoolean());
         }
         public DataParameter<Boolean> func_187161_a(int p_187161_1_)
         {
@@ -145,15 +145,15 @@
     };
     public static final DataSerializer<Rotations> field_187199_i = new DataSerializer<Rotations>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Rotations p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Rotations value)
         {
-            p_187160_1_.writeFloat(p_187160_2_.func_179415_b());
-            p_187160_1_.writeFloat(p_187160_2_.func_179416_c());
-            p_187160_1_.writeFloat(p_187160_2_.func_179413_d());
+            buf.writeFloat(value.func_179415_b());
+            buf.writeFloat(value.func_179416_c());
+            buf.writeFloat(value.func_179413_d());
         }
-        public Rotations func_187159_a(PacketBuffer p_187159_1_)
+        public Rotations func_187159_a(PacketBuffer buf)
         {
-            return new Rotations(p_187159_1_.readFloat(), p_187159_1_.readFloat(), p_187159_1_.readFloat());
+            return new Rotations(buf.readFloat(), buf.readFloat(), buf.readFloat());
         }
         public DataParameter<Rotations> func_187161_a(int p_187161_1_)
         {
@@ -162,13 +162,13 @@
     };
     public static final DataSerializer<BlockPos> field_187200_j = new DataSerializer<BlockPos>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, BlockPos p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, BlockPos value)
         {
-            p_187160_1_.func_179255_a(p_187160_2_);
+            buf.func_179255_a(value);
         }
-        public BlockPos func_187159_a(PacketBuffer p_187159_1_)
+        public BlockPos func_187159_a(PacketBuffer buf)
         {
-            return p_187159_1_.func_179259_c();
+            return buf.func_179259_c();
         }
         public DataParameter<BlockPos> func_187161_a(int p_187161_1_)
         {
@@ -197,13 +197,13 @@
     };
     public static final DataSerializer<EnumFacing> field_187202_l = new DataSerializer<EnumFacing>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, EnumFacing p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, EnumFacing value)
         {
-            p_187160_1_.func_179249_a(p_187160_2_);
+            buf.func_179249_a(value);
         }
-        public EnumFacing func_187159_a(PacketBuffer p_187159_1_)
+        public EnumFacing func_187159_a(PacketBuffer buf)
         {
-            return (EnumFacing)p_187159_1_.func_179257_a(EnumFacing.class);
+            return (EnumFacing)buf.func_179257_a(EnumFacing.class);
         }
         public DataParameter<EnumFacing> func_187161_a(int p_187161_1_)
         {
@@ -212,18 +212,18 @@
     };
     public static final DataSerializer<Optional<UUID>> field_187203_m = new DataSerializer<Optional<UUID>>()
     {
-        public void func_187160_a(PacketBuffer p_187160_1_, Optional<UUID> p_187160_2_)
+        public void func_187160_a(PacketBuffer buf, Optional<UUID> value)
         {
-            p_187160_1_.writeBoolean(p_187160_2_.isPresent());
+            buf.writeBoolean(value.isPresent());
 
-            if (p_187160_2_.isPresent())
+            if (value.isPresent())
             {
-                p_187160_1_.func_179252_a((UUID)p_187160_2_.get());
+                buf.func_179252_a((UUID)value.get());
             }
         }
-        public Optional<UUID> func_187159_a(PacketBuffer p_187159_1_)
+        public Optional<UUID> func_187159_a(PacketBuffer buf)
         {
-            return !p_187159_1_.readBoolean() ? Optional.<UUID>absent() : Optional.of(p_187159_1_.func_179253_g());
+            return !buf.readBoolean() ? Optional.<UUID>absent() : Optional.of(buf.func_179253_g());
         }
         public DataParameter<Optional<UUID>> func_187161_a(int p_187161_1_)
         {
